traffic-access-control.md

```diff
@@ -73,6 +73,49 @@ sources:
 - kind: ServiceAccount
   name: prometheus
   namespace: default
+- kind: Service
```

Comment:
In the main write-up you mentioned about supporting both ServiceAccount and service. Users can use what they want. Can users use only service name even for single K8S clusters?